invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -68,4 +68,9 @@
     width: unset;  // Allows the sidebar cards to take the full space of their column
   }
 
+  .ui.list .disabled {
```

Comment:
This can be easily moved to `list.overrides` so list components can use it along the instance :)

Reply:
done!